dali/dfuplus/dfuplus.cpp

```diff
@@ -367,15 +367,29 @@ bool CDfuPlusHelper::variableSpray(const char* srcxml,const char* srcip,const ch
 
     const char* encoding = globals->queryProp("encoding");
     const char* rowtag = globals->queryProp("rowtag");
-    if(stricmp(format, "xml") == 0)
+    const char* rowpath = globals->queryProp("rowpath");
+    if(strieq(format, "json"))
```

Comment:
Is there any special reason to use strieq() function here? (Further down there is stricmp() used in this condition chain. Not a big deal only curiosity.)